### ISSUE TITLE
Searching: country/part

### ISSUE THREAD

**Issue by ErwinKomen** [ErwinKomen/RU-stalla#29]
> De combinatie ‘Land’ en ‘Onderdeel’ zoeken werkt niet goed. Waarschijnlijk komt dat omdat er categorieën in staan die wij niet in de database hebben, bijvoorbeeld ‘Choir stall (Wang)’

**Comment by ErwinKomen** [comment]
1. The category 'Choir stall (Wang)' is in object 02277
    1. Note: this is the combination of the English name ('Choir stall') with the Dutch name ('Wang'), something that is needed to distinguish some of the categories from others that have the same name in English, but a different one in Dutch (or the reverse)
2. I cannot find a problem in the combination.
Here's an example of a possibility: Location **Danmark**, Part **Knop** (in its 4 variants)
![issue29_a](https://user-images.githubusercontent.com/11887269/152775163-1acc855d-0108-43a5-9d96-f03c2890e3b1.png)


As for the 'variants' in 'onderdeel', here's for instance what the database gives for 'knop':

id | eng | naam
--- | --- | ---
"2" | "Armrest (false)" | "Knop (reliëf)"
"7" | "Armrest" | "Knop"
"36" | "Bench-end" | "Knop"
"43" | "Bench-end" | "Knop (reliëf)"

**Comment by ErwinKomen** [comment]
B.t.w, this is the total content of `Soort` as available inside the Access database `object` via this query:
```
SELECT DISTINCT object.Soort, object.Soort_engels
FROM [object]
ORDER BY object.Soort, object.Soort_engels;
```


<HTML>
<HEAD>
<META HTTP-EQUIV="Content-Type" CONTENT="text/html;charset=UTF-8">
</HEAD>
<BODY>
<TABLE BORDER=1 BGCOLOR=#ffffff CELLSPACING=0><FONT FACE="Calibri" COLOR=#000000><CAPTION><B>qSoort</B></CAPTION></FONT>

<THEAD>
<TR>
<TH BGCOLOR=#c0c0c0 BORDERCOLOR=#000000 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Soort</FONT></TH>
<TH BGCOLOR=#c0c0c0 BORDERCOLOR=#000000 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Soort_engels</FONT></TH>

</TR>
</THEAD>
<TBODY>
<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000><BR></FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Bench-end</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Armleuning</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Armrest</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Armleuning</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Elbowrest</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Armleuning (beeld)</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Console</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Armleuning (beeld)</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Partition (sculpture)</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Bank</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Stall</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Console</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Console</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Console (reli&#235;f)</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Console (relief)</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Cul-de-lampe</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Cul-de-lampe</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Dorsaal</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Dorsal</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Dorsaal (detail)</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Canopy (detail)</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Dorsaal (detail)</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Dorsal (detail)</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Dorsaal (medaillon)</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Dorsal (medallion)</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Dorsaal (reli&#235;f)</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Dorsal (relief)</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Knop</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Armrest</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Knop</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Bench-end</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Knop (reli&#235;f)</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Armrest (false)</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Knop (reli&#235;f)</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Bench-end</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Koorbank</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Bench-end</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Koorbank</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Choir stall</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Koorbank (achterzijde)</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Choir stall (back)</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Koorbank (achterzijde)</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Misericord</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Lezenaar</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Lectern</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Lezenaar</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Reading desk</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Lezenaar (dekstuk)</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Dorsal (relief)</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Lezenaar (dekstuk)</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Reading desk (top)</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Lezenaar (reli&#235;f)</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Reading desk (relief)</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Lezenaar (zijkant)</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Reading desk (side panel)</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Misericorde</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Choir stall</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Misericorde</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Misericord</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>n.v.t.</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>n/a</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Onbekend</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Unknown</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Onderpaneel</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Lower back panel</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Overhuiving</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Bench-end</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Overhuiving</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Canopy</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Overhuiving (detail)</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Canopy (detail)</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Overig</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Other</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Pendentief</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Pendant boss</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Rugleuning</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Seat back</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Tussenschot</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Partition</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Tussenschot</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Partition (sculpture)</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Tussenschot (dekstuk)</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Partition (top)</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Vierpassen</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Quatrefoils</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Wang</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Bench</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Wang</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>bench end</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Wang</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Bench-end</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Wang</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Choir stall</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Wang</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Wang</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Wang (dekstuk)</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Bench-end</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Wang (dekstuk)</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Bench-end (detail)</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Wang (dekstuk)</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Bench-end (top)</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Wang (dekstuk)</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Choir stall (top)</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Wang (detail)</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Bench-end</FONT></TD>

</TR>

<TR VALIGN=TOP>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Wang (detail)</FONT></TD>
<TD BORDERCOLOR=#d0d7e5 ><FONT style=FONT-SIZE:11pt FACE="Calibri" COLOR=#000000>Bench-end (detail)</FONT></TD>

</TR>
</TBODY>
<TFOOT></TFOOT>
</TABLE>
</BODY>
</HTML>

**Comment by ErwinKomen** [comment]
The database has been cleaned up to prevent duplicates.
I have now added code to update the `Soort` table at the end of performing a JSON update of the database.
This means that it is now possible to show just the appropriate language for Soort.

There was a variant `wang` instead of `Wang` (case problem). 
Resolution: I adapted `Werkstuk` method `custom_set()` to do a case-independant search for soort.
Redoing synchronisation one more time...

Okay, this worked.
